docs/How to write impls.md

```diff
@@ -0,0 +1,242 @@
+---
+author: Joseph Cook (@jmcook1186)
+abstract: Guidance for writing valid impls.
+---
+
+# IMPL writing guide
+
+The Impact Framework receives all its configuration and observation data in the form of a `yaml` file known as an `impl` (input-yaml).
+To use the framework, a user only has to write an `impl` file and pass its path to the command line tool. This guide will help you to understand how to construct an `impl` you can use to measure the energy and carbon usage of your app.
```

Comment:
pass it's

Reply:
how to construct an `impl`, use it for measuring the energy and carbon usage of your app.

Reply:
The apostrophe does not belong there. Rewording second half as suggested.

```suggestion
To use the framework, a user only has to write an `impl` file and pass its path to the command line tool. This guide will help you to understand how to construct an `impl` and use it to measure the energy and carbon usage of your app.
```